ui/src/components/UserMenu/index.jsx

```diff
@@ -106,14 +107,18 @@ export default class UserMenu extends Component {
           anchorEl={anchorEl}
           open={Boolean(anchorEl)}
           onClose={this.handleMenuClose}>
+          <MenuItem title="Your Dashboard" component={Link} to="/">
+            <HomeAnalyticsIcon className={classes.leftIcon} />
+            Dashboard
+          </MenuItem>
```

Comment:
No need for that. We rely on the convention of clicking the logo to go back to the home page. Sometimes, we only want users to view the documentation site (e.g., https://docs.taskcluster.net).